benches/records.rs

```diff
@@ -0,0 +1,44 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use pprof::criterion::{Output, PProfProfiler};
+use utilities::{bench_args, EvalMode};
+
+fn count_letters(c: &mut Criterion) {
+    bench_args(
+        "countLetters",
+        env!("CARGO_MANIFEST_DIR"),
+        "records/countLetters",
+        None,
+        vec![String::from(
+            r#"
+            "
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sed vulputate mi sit amet. Et odio pellentesque diam volutpat. Pharetra et ultrices neque ornare. Aenean pharetra magna ac placerat vestibulum lectus mauris ultrices. Iaculis nunc sed augue lacus viverra vitae congue eu. Semper eget duis at tellus at. Elementum sagittis vitae et leo duis ut diam quam nulla. Nisl pretium fusce id velit ut. Magnis dis parturient montes nascetur. Leo duis ut diam quam nulla porttitor massa. Non diam phasellus vestibulum lorem sed risus. Adipiscing vitae proin sagittis nisl rhoncus mattis. Imperdiet dui accumsan sit amet nulla. Venenatis a condimentum vitae sapien pellentesque habitant morbi tristique senectus.""#, // Sollicitudin ac orci phasellus egestas tellus rutrum tellus pellentesque. Venenatis cras sed felis eget velit. Tortor pretium viverra suspendisse potenti nullam ac. Pretium vulputate sapien nec sagittis aliquam malesuada. Felis eget velit aliquet sagittis id consectetur. Non consectetur a erat nam at lectus urna. Habitant morbi tristique senectus et netus. Praesent semper feugiat nibh sed. Maecenas accumsan lacus vel facilisis volutpat est velit egestas dui. Facilisi etiam dignissim diam quis. Vitae congue eu consequat ac felis donec. Maecenas sed enim ut sem viverra aliquet eget sit. Pulvinar etiam non quam lacus suspendisse faucibus. Donec pretium vulputate sapien nec sagittis aliquam malesuada. Aliquam sem fringilla ut morbi tincidunt augue interdum. Urna duis convallis convallis tellus id interdum velit laoreet.
```

Comment:
We may want to put this in a separate file and include it at compile time, as we do for the the stdlib with `include_str!` (I'm not sure about the exact name)